ledger/sandbox/src/main/scala/com/digitalasset/platform/index/StandaloneIndexerServer.scala

```diff
@@ -11,9 +12,12 @@ import scala.concurrent.duration._
 // Main entry point to start an indexer server.
 // See v2.ReferenceServer for the usage
 object StandaloneIndexerServer {
+  private[this] val actorSystem = ActorSystem("StandaloneIndexerServer")
```

Comment:
Let's keep this `actorSystem` here and pass it on to the `PostgresIndexer` instead of creating another one there.